### PR TITLE
Update french translations

### PR DESCRIPTION
**Pull request by dacmot**

### CODE REVIEW

File: mods/default/locale/default.fr.tr [hunk 1]
```diff
@@ -67,12 +67,12 @@ Obsidian Block=Block d'obsidienne
...
 Dirt with Grass=Terre avec de l'herbe
 Dirt with Grass and Footsteps=Terre avec de l'herbe et des traces de pas
-Dirt with Savanna Grass=
+Dirt with Savanna Grass=Terre avec de l'herbe de savanne
 Dirt with Snow=Terre avec de la neige
 Dirt with Rainforest Litter=Terre avec sol de forêt humide
 Dirt with Coniferous Litter=Terre avec sol de forêt de conifère
-Savanna Dirt=
-Savanna Dirt with Savanna Grass=
+Savanna Dirt=Terre de la savane
```
Comment (appgurueu) [inline mods/default/locale/default.fr.tr:74]: Typo: Should be `savanne`?

**Review by appgurueu — APPROVED** [review]
Looks fine to me except for a typo.

File: mods/default/locale/default.fr.tr (hunk 1 above)
Comment (dacmot) [reply mods/default/locale/default.fr.tr:74]: Actually, it should be `savane`... I've changed it everywhere in the following commit.

**Review by Df458 — APPROVED** [review]
I'm not entirely fluent in French I can't confirm how natural these are, but they seem fine to me.

File: mods/mtg_craftguide/locale/mtg_craftguide.fr.tr [hunk 2]
```diff
@@ -30,24 +31,24 @@ Any magenta dye=Quelconque colorant magenta
...
+Unknown Item=Object inconnu
+Fuel=Combustible
 Usage @1 of @2=Usage @1 sur @2
 Recipe @1 of @2=Recette @1 sur @2
-Previous recipe=
-Next recipe=
-Recipe is too big to be displayed.=
+Previous recipe=Recette précédente
+Next recipe=Prochaine recette
+Recipe is too big to be displayed.=La recette est trop grande pour être affichée.
```
Comment (Calinou) [inline mods/mtg_craftguide/locale/mtg_craftguide.fr.tr:42]: Shorten the translation a bit to make sure it fits on small displays or with large font sizes:

```suggestion
Recipe is too big to be displayed.=Recette trop grande pour être affichée.
```

**Review by Calinou — APPROVED** [review]
I proofread the changed strings and the translation seems good to me, only one suggestion to look at.

**Pull request merged**